rest_api_routines.sh

```diff
@@ -57,7 +57,22 @@ function callPOSTService {
     fi
 
     echo "Calling URI (POST): " ${uri}
-    curl -X POST -H "${HEADER_ACCEPT}" -H "${HEADER_CONTENT_TYPE}" -u "${USER_NAME}:${USER_PASSW}" "$certAtt" "${ENGINE_URL}${uri}" -d "${xml}" 2> /dev/null > "${COMM_FILE}"
+    echo "${ENGINE_URL}${uri}"
+    curl -X POST -H "${HEADER_ACCEPT}" -H "${HEADER_CONTENT_TYPE}" -u "${USER_NAME}:${USER_PASSW}" -k "$certAtt" "${ENGINE_URL}${uri}" -d "${xml}" -L 2> /dev/null> "${COMM_FILE}"
```

Comment:
Well, there would be 2 options concurently now - one is to use trusted certificate and the other is to use insecure connection (ignore trusted CA certificate). And from the documentation is not clear, which is prefered. I tested it, it works, I can use both options in paralel, but I am not sure which is better, so maybe I would modify the previous code:

if [[ -n "$CA_CERT_PATH" ]]; then
        certAtt="--cacert $CA_CERT_PATH"
else
        certAtt="-k"
fi